Jenkinsfile

```diff
@@ -23,7 +23,8 @@ elifePipeline {
                 },
                 'test': {
                     withCommitStatus({
-                        sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_app_test app npm test"
+                        sh "vault.sh kv get -field token secret/containers/elife-xpub/codecov > .codecov-token"
+                        sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run -e CODECOV_TOKEN=$(cat .codecov-token) --rm --name elife-xpub_app_test app npm run test-ci"
```

Comment:
`$(...)` should be `${...}`

Reply:
or maybe not 🤷‍♂️

Reply:
`${...}` is a (Groovy) substitution of a variable into the string while the Jenkinsfile gets executed.
`$(...)` is a (bash) substitution of a command's result into the string while the command in `sh` gets executed.